troupon/static/scss/partials/_sections.scss

```diff
@@ -397,3 +397,28 @@ footer {
     margin-top: 30px;
   }
 }
+
+// messages
+.strip {
+  padding: 10px 5px 10px;
+}
+.subject {
+  font-size: 22px;
+  padding-bottom: 5px;
+}
+#compose-form {
+  padding-bottom: 8px;
+}
+.message {
+  background: #ffffff;
+  padding-top: 5px;

```

Comment:
Properties should be ordered background, margin-bottom, margin-top, padding-bottom, padding-top